core/src/main/java/org/apache/accumulo/core/util/LocalityGroupUtil.java

```diff
@@ -50,16 +52,16 @@
 
   // private static final Logger log = Logger.getLogger(ColumnFamilySet.class);
 
-  public static final Set<ByteSequence> EMPTY_CF_SET = Collections.emptySet();
+  public static final Set<ByteSequence> EMPTY_CF_SET = ImmutableSet.of();
 
   public static Set<ByteSequence> families(Collection<Column> columns) {
     if (columns.size() == 0)
       return EMPTY_CF_SET;
-    Set<ByteSequence> result = new HashSet<>(columns.size());
+    Builder<ByteSequence> builder = ImmutableSet.builder();
     for (Column col : columns) {
-      result.add(new ArrayByteSequence(col.getColumnFamily()));
+      builder.add(new ArrayByteSequence(col.getColumnFamily()));
     }
-    return result;
+    return builder.build();
```

Comment:
Could make this a bit more concise with a Java8 lambda since this is slated for master.